include/seqan3/argument_parser/validators.hpp

```diff
@@ -172,17 +174,17 @@ class arithmetic_range_validator
 /*!\brief A validator that checks whether a value is inside a list of valid values.
  * \ingroup argument_parser
  * \implements seqan3::validator
- * \tparam option_value_t Must be a (container of) arithmetic type(s).
+ * \tparam option_value_t \copybrief seqan3::option_value_type::option_value_type
  *
  * \details
  *
  * On construction, the validator must receive a range or parameter pack of valid values.
  * The class than acts as a functor, that throws a seqan3::validation_error
  * exception whenever a given value is not in the given list.
  *
- * \note In order to simplify the chaining of validators, the option value type is deduced to `double` for ranges whose
- *       value type models seqan3::arithmetic, and to `std::string` if the ranges value type is convertible to it.
- *       Otherwise, the option value type is deduced to the value type of the range.
+ * \note In order to simplify the chaining of validators, the option value type is deduced to `std::string` if the
+ *       ranges value type is convertible to it. Otherwise, the option value type is deduced to the value type of the
+ *       range.
```

Comment:
```suggestion
 * \note In order to simplify the chaining of validators, the option value type is deduced to `std::string` if the
 *       range's value type is convertible to it. Otherwise, the option value type is deduced to the value type of the
 *       range.
```